lib/common.sh

```diff
@@ -204,6 +206,8 @@ export BUILD_BMO_LOCALLY="${BUILD_BMO_LOCALLY:-false}"
 export BUILD_CAPI_LOCALLY="${BUILD_CAPI_LOCALLY:-false}"
 export BUILD_IRONIC_IMAGE_LOCALLY="${BUILD_IRONIC_IMAGE_LOCALLY:-false}"
 export BUILD_MARIADB_IMAGE_LOCALLY="${BUILD_MARIADB_IMAGE_LOCALLY:-false}"
+# FIXME(dtantsur): flip to false once everything merges
+export BUILD_IRSO_LOCALLY="${BUILD_IRSO_LOCALLY:-true}"
```

Comment:
Just noting that dev-env best practice would be to set this to value it should be and then put required test config on top of `config_example.sh`.  This way it is clear what is prod code and what is test code.

Reply:
Good to know, noted.